specification/machinelearningservices/resource-manager/readme.md

```diff
@@ -46,7 +46,39 @@ suppressions:
     reason: Suppress as instructed, this patch is for a abstract class and the type-discriminator needs to be required.
     where:
       - $.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/connections/{connectionName}"].patch.parameters[5].schema.properties.properties
-```
+  - code: ResourceNameRestriction
+    reason: Experience is the same as previous GA version, adding restriction will be a breaking change.
+    where:
+      - $.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/computes/{computeName}"]
```

Comment:
why did you name this resource type 'computes'? (probably not the question you're hoping for! but...)

Reply:
the resource can be all type of compute resource use can select and use.